Makefile

```diff
@@ -108,8 +108,10 @@ clean: ## clean up binaries
 	@echo "🐳 $@"
 	@rm -f $(BINARIES)
 
-install: $(BINARIES) ## install binaries
-	@echo "🐳 $@"
+install: ## install binaries
+	@ls $(BINARIES) > /dev/null 2>&1 || \
```

Comment:
Is using "ls" really going to give you what you want since that just generates the error when they're missing, not when they're out of date?

Would something like this instead help:
```
install: testBins
	@echo "🐳 $@"
	@mkdir -p $(DESTDIR)/bin
	@install $(BINARIES) $(DESTDIR)/bin

testBins:
	@for i in $(COMMANDS); do \
		[ cmd/$$i -ot bin/$$i ] && continue ; \
		echo rebuild please ; false ; \
	done
```